c_binding/include/libstoragemgmt/libstoragemgmt_plug_interface.h

```diff
@@ -1030,6 +1031,138 @@ struct lsm_ops_v1_2 {
+ * Used to retrieve the plugin-private data for a specific battery.
+ * @param b     Battery to retrieve plugin private data for
+ * @return NULL if doesn't exists, else data.
+ */
+const char LSM_DLL_EXPORT *lsm_battery_plugin_data_get(lsm_battery *b);
+
+/**
+ * Query the RAM cache information of a volume
+ * @param[in]   c               Valid lsm plug-in pointer
+ * @param[in]   volume          Volume to be deleted
+ * @param[out]  write_cache_policy
+ *                              The write cache policy.
+ * @param[out] write_cache_status
+ *                              The status of write cache.
+ * @param[out] read_cache_policy
+ *                              The policy for read cache.
+ * @param[out] read_cache_status
+ *                              The status of read cache.
+ * @param[out] physical_disk_cache
+ *                              Whether physical disk's cache is enabled or not.
+ * @param[in]   flags           Reserved
+ * @return LSM_ERR_OK, else error reason
+ */
+typedef int (*lsm_plug_volume_cache_info) (lsm_plugin_ptr c,
+                                           lsm_volume *volume,
+                                           uint32_t *write_cache_policy,
+                                           uint32_t *write_cache_status,
+                                           uint32_t *read_cache_policy,
+                                           uint32_t *read_cache_status,
+                                           uint32_t *physical_disk_cache,
+                                           lsm_flag flags);
+
+/**
+ * New in version 1.3.
+ * Change the setting of RAM physical disk cache of certain volume.
+ * @param[in] c                   Valid lsm plug-in pointer
+ * @param[in] volume              Volume to be changed.
+ * @param[in] pdc                 Physical disk cache setting.
+ * @param[in] flags               Reserved
+ * @return LSM_ERR_OK, else error reason
+ */
+typedef int (*lsm_plug_volume_physical_disk_cache_set) (lsm_plugin_ptr c,
+                                                        lsm_volume *volume,
+                                                        uint32_t pdc,
+                                                        lsm_flag flags);
+/**
+ * New in version 1.3.
+ * Change the setting of RAM write disk cache policy of certain volume.
+ * @param[in] c                   Valid lsm plug-in pointer
+ * @param[in] volume              Volume to be changed.
+ * @param[in] wcp                 Write disk cache policy.
+ * @param[in] flags               Reserved
+ * @return LSM_ERR_OK, else error reason
+ */
+typedef int (*lsm_plug_volume_write_cache_policy_set) (lsm_plugin_ptr c,
+                                                       lsm_volume *volume,
+                                                       uint32_t wcp,
+                                                       lsm_flag flags);
+
+/**
+ * New in version 1.3.
+ * Change the setting of RAM read disk cache policy of certain volume.
```

Comment:
@tasleson mentioned this in a couple other comments, but I want to +1 the change to "specific" instead of "certain". There are many instances across these headers that need up be updated that haven't been specifically called out.